src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.subsystems.intake;
+
+import java.util.TreeMap;
+
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkAbsoluteEncoder.Type;
+
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class Intake extends SubsystemBase {
+
+    private final CANSparkMax pivotMotor = new CANSparkMax(RobotMap.INTAKE_PIVOT_MOTOR_CAN_ID, MotorType.kBrushless);
+    private final RelativeEncoder pivotEncoder = pivotMotor.getEncoder();
+    private final AbsoluteEncoder pivotAbsoluteEncoder = pivotMotor.getAbsoluteEncoder(Type.kDutyCycle);
+    private final TalonFX intakeTalonFront = new TalonFX(RobotMap.INTAKE_TALON_FRONT_CAN_ID);
```

Comment:
We should configure everything with this. There's a lot of kinds of configuration https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/configs/TalonFXConfiguration.html